### DISCUSSION TITLE
Add new RLS Policy template for insert

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#16990] [Contribute to Supabase]
Create a new RLS policy template for insert based on user_id. 
We currently have one for delete but not for insert, which is also extremely common:

<img width="1203" alt="CleanShot 2023-06-28 at 11 45 40@2x" src="https://github.com/supabase/supabase/assets/105593/12bf5744-0146-4830-9703-deb84ab629c0">

**Comment by ItIsOHM** [comment]
Hey there, I'd love to work on this issue! Can you please assign it to me and point me in the right direction as to where I can fix this issue from? :D

**Comment by weykon** [comment]
Hi, I'm now forking the repository to try it out now, if I succeed I'll raise a PR here

**Comment by a0m0rajab** [comment]
@ItIsOHM @weykon I just opened a PR for that, feel free to check the edit if you are interested in the approach.

What I did to solve this was: 
1 - searched the example provided in the image 
2- when I found the text I edited the related template and added one.

**Comment by ItIsOHM** [comment]
> @ItIsOHM @weykon I just opened a PR for that, feel free to check the edit if you are interested in the approach.
> 
> What I did to solve this was: 1 - searched the example provided in the image 2- when I found the text I edited the related template and added one.

Ohh alright, thanks for the help @a0m0rajab! :D

**Comment by a0m0rajab** [comment]
Why this issue moved into a discussion?
